R/linear_pool.R

```diff
@@ -29,6 +33,10 @@
 #'
 #' Steps 1 and 2 in this process are performed by `distfromq::make_q_fn`.
 #'
+#' When the `output_type` is `sample`, we randomly select a subset of the sample
+#' forecasts from each of the component models so that the ensemble has the
+#' specified number of `n_output_samples`.
+#'
```

Comment:
Since we removed this functionality, maybe we should remove this documentation for now, and plan to add it back in when we add this feature (soon).
```suggestion
```